ui/packages/consul-ui/app/components/consul/bucket/list/README.mdx

```diff
@@ -36,6 +55,7 @@ At the time of writing, this is not currently used across the entire UI
 | `item` | `array` |  | A Consul object that could have both a `Partition` and a `Namespace` property |
 | `nspace` | `string` | | The name of the current namespace |
 | `partition` | `string` | | The name of the current partition |
+| `service` | `boolean` | | Whether to show the service name on the end or not. Please note you must also pass a item.Service for it to show. We flag this incase an API request has a Service property but you don't want to show it |
```

Comment:
Very minor take-it-or-leave-it suggestion just to prove I’ve read the docs:

Could we name this argument a little more explicitly? Maybe `showServiceName` rather than just `service`? My rationale it that its two sibling arguments (`namespace` and `partition`) both take strings and so my initial assumption was that `service` would take a service name string.

Reply:
Yeah I get your thinking here, I wish all the arguments worked/were shaped the same also, I think its half the reason I spent a bit of extra effort on the docs and called it out in the PR, if I'm not happy with something myself, I'll generally call it out for ideas.

Opposing viewpoint here is: I'm always cautious of more than simple or commonly used argument names as they have a different problem of remembering them, or mistyping them.

I think the other alternative would be change it to accept the name of the service itself and if its set then use that, then we keep the super simple `service` lowercased argument names which is somewhat of a common thing in the codebase 🤔

Let me think a bit (or let me know if you'd prefer one over the other if we have two options). Typically this option has the downside of not working like the others which are kind of a "if its not this exact specified name then show the thing", whereas the service one would be just "show this thing"

I'd like to change it just not sure which way to go.